packages/services/f-wdio-utils/.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+dist
+page.object.js
```

Comment:
I don't think the page object should be gitignored?

Reply:
oh yeah so yarn lint doesn't seem to like the page.object - keeps coming back with `UnhandledPromiseRejectionWarning: Unhandled promise rejection.` Help? 😆

Reply:
As discussed, I mistook this file for .gitignore. Ignore my comment :)